### PR TITLE
Removed dependency for translatable module

### PR DESCRIPTION
**Pull request by micmania1**
To go with https://github.com/silverstripe/silverstripe-translatable/pull/231

### CODE REVIEW

File: code/controllers/CMSPageAddController.php [hunk 1]
```diff
@@ -148,9 +148,7 @@ public function doAdd($data, $form) {
 		}
 
 		$record = $this->getNewItem("new-$className-$parentID".$suffix, false);
-		if(class_exists('Translatable') && $record->hasExtension('Translatable') && isset($data['Locale'])) {
-			$record->Locale = $data['Locale'];
-		}
+		$this->extend('updateDoAdd', $record, $data);
```
Comment (dhensby) [inline code/controllers/CMSPageAddController.php]: passing the form (and not data) would be better

**Comment by dhensby** [comment]
happy for merge when green ticks

**Comment by dhensby** [comment]
Why is behat failing :(

**Comment by micmania1** [comment]
Can you restart the tests? They don't seem related and I have has random test failures in the past.

**Comment by dhensby** [comment]
restarted

**Comment by tractorcow** [comment]
Restarted again... cms tests have been weird lately. Will investigate soonish.

**Pull request merged**